### DISCUSSION TITLE
Closing lid laptop behaviour

### DISCUSSION THREAD

**Discussion by andreichalapco** [i3/i3#5488] [Q&A] [answered]
Is there a way in i3 when closing the lid of the laptop to disable the output of the laptop monitor but not the hdmi output?

This is possible in sway with 
```
set $laptop <laptop-output-name-or-id>
bindswitch lid:on output $laptop disable
bindswitch lid:off output $laptop enable
```

**Comment by okraits** [comment]
No, there isn't. You can do it like you would do with any other X11 window manager.

**Comment by orestisfl** [comment, marked as answer]
You can probably use [autorandr](https://github.com/phillipberndt/autorandr) for this. See this comment: https://github.com/phillipberndt/autorandr/issues/104#issuecomment-467361778
